app/pages/amigo-requests/amigo-requests.html

```diff
@@ -7,4 +7,6 @@
     <amigo-share-request-card *ngFor="let amigoShareRequest of amigoShareRequests"
                               [amigoShareRequest]="amigoShareRequest"></amigo-share-request-card>
 
-</ion-content>
+    <amigo-request-fab></amigo-request-fab>
+
+</ion-content>
```

Comment:
Example of FAB:

<img width="378" alt="fab" src="https://cloud.githubusercontent.com/assets/6886583/16640182/e4550874-43c3-11e6-87d4-ac84f9e77cb2.png">